chukwuemekachm/M-Tracker-React-Client
# #160920323 Completes filter request feature

#### What does this PR do?
- This PR completes filter request feature
#### Description of Task to be completed?
- Add filterRequestsAction.js
- Modify Dashboard.jsx
- Modify SideNav.jsx
- Update existing snapshots
- Write tests for feature
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Navigate to the project's directory
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser will be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- Click on any of the filter buttons on the side nav
- Observe the behaviour of the application
#### Production url
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160920323](https://www.pivotaltracker.com/story/show/160920323)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-02 at 6 33 44 pm" src="https://user-images.githubusercontent.com/33798252/46371047-2f35c780-c67f-11e8-9555-4dbf72dcadb3.png">


<img width="1440" alt="screen shot 2018-10-02 at 6 33 54 pm" src="https://user-images.githubusercontent.com/33798252/46371048-2f35c780-c67f-11e8-8205-8a50f152ba62.png">

#### Questions:
- None
